portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,20 @@
   </head>
   <body>
     <div id="content">
-    <h1>My Portfolio</h1>
-    <p>Hi! I'm Lily, a STEP Intern at Google this summer and a rising junior @ Stanford studying CS + English.</p>
-    <img src="/images/flower.jpg" alt="Red Flower">
-    <p>Click here to get a random message:</p>
-    <button onclick="addRandomGreeting()">Hello</button>
-    <div id="greeting-container"></div>
+      <h1>LILY ZHOU</h1>
```

Comment:
Just FYI, no need to change anything if you don't want, there's a css property called `text-transform` that lets you change the rendered capitalization of the enclosed text to be all uppercase, first letter uppercase, or all lowercase.

It could be useful if you were maintaining a large site and wanted the option to change all of the headings to be all uppercase with a single line of CSS, or you could use it with [@media rules](https://developer.mozilla.org/en-US/docs/Web/CSS/@media) to make it uppercase on screen and lowercase when printed. I don't know why you'd want to do that, but a more practical equivalent might be to change the font or font size to be more legible when printed.